_posts/2015-01-08-thank-you-open-source-community.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: "Thank you, open-source community"
+author: Ayaka Nonaka
+---
+
+We open-source a lot of things at Venmo, and we also use a lot of open-source libraries. To show some appreciation to all of the open-source maintainers and contributors out there, we thought it might be cool to share a list of open-source libraries and tools that the Venmo iOS team relies on. Thank you everyone out there who has contributed to these projects and other projects out there. Here’s to a great 2015!
+
+* [1PasswordExtension](https://github.com/AgileBits/onepassword-app-extension) adds 1Password support to our login. (Psssst... Look out for it in our next release. ;))
+* [AFNetworking](https://github.com/AFNetworking/AFNetworking) is great for all things networking, but we especially like `UIImageView+AFNetworking` for async image loading.
+* [Alcatraz](https://github.com/supermarin/Alcatraz) is the package manager that brings XVim to Xcode, and that makes me happy.
+* [BZGFormViewController](https://github.com/benzguo/BZGFormViewController) is great for simple forms that require validation. We use it in for in our sign up and edit profile views.
+* [BZGMailgunEmailValidation](https://github.com/benzguo/BZGMailgunEmailValidation) is perfect if you use Mailgun for email validation.
+* [BlocksKit](https://github.com/zwaldowski/BlocksKit) because `dismissWithClickedButtonIndex:animated:` delegate methods are no fun at all. Besides, who “clicks” on an iOS device?
+* [Braintree](https://github.com/braintree/braintree_ios) allows our users to pay via Venmo on apps like [YPlan](https://yplanapp.com/).
+* [CMPopTipView](https://github.com/chrismiles/CMPopTipView) has been useful when we want to draw attention to a new feature that we added.
+* [CocoaPods](https://github.com/cocoapods/cocoapods) is how we manage all of our dependencies, including private pods. Can’t wait for it to support Swift.
+* [DAZABTest](https://github.com/dasmer/DAZABTest) provides a super simple API to do basic A/B tests.
+* [Expecta](https://github.com/specta/expecta) is a great matcher framework that makes your tests read like English. `expect(myTests).toNot.beEmpty()`
+* [FLEX](https://github.com/Flipboard/FLEX) is built into all of our debug and dogfood builds, and it makes finding and fixing UI bugs so much fun and so much easier.
```

Comment:
I guess we can add Facebook SDK too, since we technically rely on it. :sweat_smile: